Astral Pathfinder/Models/Text.hpp

```diff
@@ -0,0 +1,46 @@
+//
+//  Text.hpp
+//  Astral Pathfinder
+//
+//  Created by Ian Holdeman on 2/24/18.
+//  Copyright © 2018 Logan Harvell, Ian Holdeman. All rights reserved.
+//
+
+#ifndef Text_hpp
+#define Text_hpp
+
+// MARK: Libraries and Frameworks
+#include "SDL2/SDL.h"
+#include "Game.hpp"
+
+// MARK: Source Files
+#include "GameObject.hpp"
+
+// MARK: - Text Parameters
+
+namespace TextParameters {
+  const SDL_Color color = {255, 255, 255};
+}
+
+
+class Text : public GameObject {
```

Comment:
May or may not have to be a GameObject ?

Reply:
I liked it as GameObject cuz being one allowed it to inherit a rect and texture and just made my life slightly easier